.github/workflows/msbuild.yml

```diff
@@ -55,11 +55,11 @@ jobs:
 
   integration-tests:
     needs: check-deep-tests
-    if: always() && ((needs.check-deep-tests.result == 'success' || contains(github.event.pull_request.labels.*.name, 'run-deep-tests')) || ( github.event_name == 'push' && ( github.ref_name == 'master' || vars.TEST_ON_FORK == 'true' )))
+    if: always() && ((needs.check-deep-tests.result == 'success' || contains(github.event.pull_request.labels.*.name, 'run-deep-tests')) || ( github.event_name == 'push' && vars.TEST_ON_FORK == 'true' ))
     uses: ./.github/workflows/integration-tests-reusable.yml
     with:
       # By default run only on one platform, but run on all platforms if the PR has the "run-deep-tests"
       # label, and skip checking the nightly build above.
       # This is the best way to fix an issue in master that was only caught by the nightly build.
-      all_platforms: ${{ contains(github.event.pull_request.labels.*.name, 'run-deep-tests') }}
+      all_platforms: ${{ contains(github.event.pull_request.labels.*.name, 'run-deep-tests') || contains(github.event.push.labels.*.name, 'run-deep-tests')}}
```

Comment:
What is the difference between these two?

Reply:
The change allows one to run deep tests on a local fork, with a push on a branch that does not have a PR created. Allows running full tests without using common resources.